lib/cisco_node_utils/cmd_ref/fabricpath.yaml

```diff
@@ -69,17 +67,23 @@ loadbalance_algorithm:
   get_command: "show run fabricpath all"
   get_value: '/^fabricpath load\-balance (\S+)\s*$/'
   set_value: "<state> fabricpath load-balance <algo>"
-  N5k:
+  N5k: &loadbal_alg_n5_n6
```

Comment:
We've used this pattern before. The reason is that it is specific to these two platforms. If we bring in another platform with yet another unique value then `loadbal_alg` isn't that meaningful.